src/solvers/dg/dg_2d_parabolic.jl

```diff
@@ -0,0 +1,189 @@
+# This file collects all methods that have been updated to work with parabolic systems of equations
+
+function rhs!(du::AbstractArray{<:Any,4}, u, gradients, t,
+              mesh::TreeMesh{2}, equations,
+              initial_condition, boundary_conditions, source_terms,
+              dg::DG, cache, cache_gradients)
+  # Reset du
+  @timeit_debug timer() "reset ∂u/∂t" du .= zero(eltype(du))
+
+  # Calculate volume integral
+  @timeit_debug timer() "volume integral" calc_volume_integral!(du, u, gradients, have_nonconservative_terms(equations), equations,
+                                                                dg.volume_integral, dg, cache)
+
+  # Prolong solution to interfaces
+  @timeit_debug timer() "prolong2interfaces" prolong2interfaces!(cache, cache_gradients, u, gradients, equations, dg)
+
+  # Calculate interface fluxes
```

Comment:
maybe put a comment here that this also resolves the viscous numerical flux as well

Reply:
Fixed in 79129395.